src/components/watermark/watermark.js

```diff
@@ -27,19 +29,14 @@ const mapStateToProps = state => ({
  * @extends {BaseComponent}
  */
 class Watermark extends BaseComponent {
-  /**
-   * @static
-   * @type {string} - Component display name
-   */
-  static displayName = 'watermark';
   /**
    * should render component
    * @param {*} props - component props
    * @returns {boolean} - whether to render the component
    * @static
    */
   static shouldRender(props: any): boolean {
-    const componentConfig = props.config.components[this.displayName];
+    const componentConfig = props.config.components['watermark'];
```

Comment:
change this and add to convention doc for exposing config containers
```suggestion
    const componentConfig = props.config.components[COMPONENT_NAME.toLowerCase()];
```
need to change in other places too where this applies

Reply:
following our discussion won't do